src/doc/data_into_visit/CurveFormat.rst

```diff
@@ -0,0 +1,13 @@
+.. _Data_Into_VisIt_Curve:
+
+the Curve file format
+=====================
+
+The curve file is a text file for specifying 1D curves.
+It consists of a curve name that is on a line starting with a `#` followed by the X-Y coordinates of the curve, one point per line.
```

Comment:
Matlib style comments (% instead of #) are also supported.
TIME and CYCLE comments are also supported  following comment with curve name:
# mycurve1
# TIME 22.5
# CYCLE 15

Reply:
Thanks for the additional information. I updated the content to incorporate it.